app/views/layouts/_header.html.erb

```diff
@@ -78,19 +78,23 @@
       <ul class="nav navbar-nav navbar-right">
         <% if current_user %>
           <li class="hidden-sm hidden-md">
+<<<<<<< HEAD
```

Comment:
Please remove the conflicts properly.
You can reset your branch to the latest version of the master.

Reply:
I've fixed the merge conflict and pushed. Do you want me to rebase and do a hard reset on this branch?

Reply:
You need to create this pr exactly same as #4203 then we can work forward. Also, you need the changes of the current master.
You can do this by pulling changes to the master. Creating a new branch integrate_modals and then copying my changes from #4203

Reply:
![image](https://user-images.githubusercontent.com/22998430/49700061-b20b5f00-fba7-11e8-9531-8c006b3ee191.png)
In form2, the log in button on the bottom links to the log in page. If I want to make the login modal pop up when that is clicked instead, the changes will also be reflected on the signup page since it also uses that component.

Should I make the signup modal load the **_form.html.erb** partial so that it won't impact the signup page?